README.md

```diff
@@ -179,7 +179,7 @@ The `source` property uses a custom selector syntax.  It’s most often seen as
 | object[^(property, ...)] | `digitalData.cart.price[^(shipping)]` | Returns an object with properties whose names begin with `property`. |
 | object[$(property, ...)] | `digitalData.page.pageInfo[$(Date)]` | Returns an object with properties whose names end with `property`. |
 | object[?(property, ...)] | `digitalData.cart[?(cartID)]` | Returns the object or null if the object does not have property. |
-| object[?(property=value, ...)] | `digitalData.cart.price[?(basePrice>=10)]` | Returns the object or null if the object's `property` does not compare to `value`. Comparison can be `=`, `<=`, `>=`, `<`, `>`, `!=`, `!^`, and `=^`. Use the value `undefined` to query if a property `=` or `!=` undefined. |
+| object[?(property=value, ...)] | `digitalData.cart.price[?(basePrice>=10)]` | Returns the object or null if the object's `property` does not compare to `value`. Comparison can be `=`, `<=`, `>=`, `<`, `>`, `!=`, `=$`, `!$`, `!^`, and `=^`. Use the value `undefined` to query if a property `=` or `!=` undefined. |
```

Comment:
Nit: In the example ordering we have `=$` and then `!$` for ends with, but `!^` and then `=^` for starts with (reversing the example order). It may make sense to be consistent with ordering between ends with and starts with operators.

Not necessary under this PR scope, but since we're accumulating a long list of example operators I wonder if it makes sense to break these out into their own table in the future -- each row with their own description - for easier consumption.